### ISSUE TITLE
Accessibility - Add tabindex '0' to all td

### ISSUE THREAD

**Issue by sreenadh** [matfish2/vue-tables-2#736]
Add tabindex '0' to all td. This would help us to navigate through td by using tab key.

**Comment by untone** [comment]
How to disable tabindexes on every td in client table? In my case, only tr cannot be selected by keyboard, but no cells. No appropriate option methods related to cell attrubures not found.

**Comment by david34corbalan** [comment]
me uno a la peticion de UNTONE como desabilitamos el tabindex??

**Comment by matfish2** [comment]
use the `tabbable` option. Set to `false`

**Comment by david34corbalan** [comment]
thanks for the reply, but attaching the options that I suggested, the tabindex is still found as an attribute. I wish it would not be so that the cell is not selectable.
https://jsfiddle.net/matfish2/jfa5t4sm/
In this link also do what is suggested and it does not work. any ideas?

**Comment by gpibarra** [comment]
I have the same problem

**Comment by matfish2** [comment]
https://jsfiddle.net/3qxmt9ag/

**Comment by gpibarra** [comment]
"tabbable:true", <td tabindex="-1" class="">Zimbabwe</td>
"tabbable:false", <td tabindex="0" class="">Zimbabwe</td>
but i want: 
<td class="">Zimbabwe</td>
For disabled selected row and make clickable all row.
